src/main/java/melky/resourcepacks/SpriteOverride.java

```diff
@@ -136,6 +140,31 @@
 	MINIMAP_ORB_WORLD_MAP_PLANET(SpriteID.MINIMAP_ORB_WORLD_MAP_PLANET, Folder.OTHER),
 	MINIMAP_ORB_WORLD_MAP_PLANET_HOVERED(SpriteID.MINIMAP_ORB_WORLD_MAP_PLANET_HOVERED, Folder.OTHER),
 
+	DIALOG_BACKGROUND(SpriteID.DIALOG_BACKGROUND, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_TOP(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_TOP, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_LEFT(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_LEFT, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_BOTTOM(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_BOTTOM, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_RIGHT(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_RIGHT, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_CORNER_TOP_LEFT(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_CORNER_TOP_LEFT, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_CORNER_TOP_RIGHT(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_CORNER_TOP_RIGHT, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_CORNER_BOTTOM_LEFT(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_CORNER_BOTTOM_LEFT, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_CORNER_BOTTOM_RIGHT(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_CORNER_BOTTOM_RIGHT, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_HORIZONTAL(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_EDGE_HORIZONTAL, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_INTERSECTION_LEFT(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_INTERSECTION_LEFT, Folder.DIALOG_BOXES),
+	BOTTOM_LINE_MODE_SIDE_PANEL_INTERSECTION_RIGHT(SpriteID.BOTTOM_LINE_MODE_SIDE_PANEL_INTERSECTION_RIGHT, Folder.DIALOG_BOXES),
+	IRON_RIVETS_VERTICAL(SpriteID.IRON_RIVETS_VERTICAL, Folder.DIALOG_BOXES),
+	IRON_RIVETS_HORIZONTAL(SpriteID.IRON_RIVETS_HORIZONTAL, Folder.DIALOG_BOXES),
+	IRON_RIVETS_CORNER_TOP_LEFT(SpriteID.IRON_RIVETS_CORNER_TOP_LEFT, Folder.DIALOG_BOXES),
+	IRON_RIVETS_CORNER_TOP_RIGHT(SpriteID.IRON_RIVETS_CORNER_TOP_RIGHT, Folder.DIALOG_BOXES),
+	IRON_RIVETS_CORNER_BOTTOM_LEFT(SpriteID.IRON_RIVETS_CORNER_BOTTOM_LEFT, Folder.DIALOG_BOXES),
+	IRON_RIVETS_CORNER_BOTTOM_RIGHT(SpriteID.IRON_RIVETS_CORNER_BOTTOM_RIGHT, Folder.DIALOG_BOXES),
+	IRON_RIVETS_EDGE_TOP(SpriteID.IRON_RIVETS_EDGE_TOP, Folder.DIALOG_BOXES),
+	IRON_RIVETS_EDGE_RIGHT(SpriteID.IRON_RIVETS_EDGE_RIGHT, Folder.DIALOG_BOXES),
```

Comment:
All of these should include the folder name at the start of them. This is done like so because there can't be duplicate enum entries. So if fixed_mode folder has edge_left and resizeable_mode has the same edge_left they wouldn't get marked as duplicates in the enum